### PR TITLE
Postgres storage fix

### PR DESCRIPTION
**Pull request by JoshLove-msft**

### CODE REVIEW

**Review by m-nash — APPROVED** [review]

File: sdk/provisioning/Azure.Provisioning/api/Azure.Provisioning.net461.cs [hunk 1]
```diff
@@ -0,0 +1,466 @@
+namespace Azure.Provisioning
```
Comment (tg-msft) [inline sdk/provisioning/Azure.Provisioning/api/Azure.Provisioning.net461.cs:1]: How is this one property making things suddenly multitarget?

File: sdk/provisioning/Azure.Provisioning/api/Azure.Provisioning.net461.cs [hunk 2]
```diff
@@ -0,0 +1,466 @@
...
+{
+    public static partial class CdkExtensions
```
Comment (tg-msft) [inline sdk/provisioning/Azure.Provisioning/api/Azure.Provisioning.net461.cs:3]: We need to rename this as soon as things calm down.

**Review by tg-msft — APPROVED** [review]

File: sdk/provisioning/Azure.Provisioning/api/Azure.Provisioning.net461.cs (hunk 2 above)
Comment (JoshLove-msft) [reply sdk/provisioning/Azure.Provisioning/api/Azure.Provisioning.net461.cs:3]: I can rename it to ProvisioningExtensions right now.

File: sdk/provisioning/Azure.Provisioning/api/Azure.Provisioning.net461.cs (hunk 1 above)
Comment (JoshLove-msft) [reply sdk/provisioning/Azure.Provisioning/api/Azure.Provisioning.net461.cs:1]: Yeah I'm not sure what happened here. We were already multitargeting, but somehow the api.cs files were not created for the other targets.

**Pull request merged**